### PR TITLE
add a example

### PR DESCRIPTION
**Pull request by ESSch**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

**Additional information**

### CODE REVIEW

**Comment by ESSch** [comment]
I add a example of remote using a kubernetes cluster

File: README.md [hunk 1]
```diff
@@ -69,6 +69,11 @@ $ docker run --rm --name kubectl bitnami/kubectl:latest -- --version
 
 Consult the [Kubectl Reference Documentation](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands) to find the completed list of commands available.
 
+Remote run:
```
Comment (alemorcuq) [inline README.md]: I would move this to the **Configuration** section, maybe in a subsection called **Load your own configuration. For example:

```markdown
## Loading your own configuration

It's possible to load your own configuration, which is useful if you want to connect to a remote cluster:

```bash
$ docker run --rm --name kubectl -v /path/to/your/kube/config:/.kube/config bitnami/kubectl:latest
```

Comment (alemorcuq) [inline README.md]: I would move this to the **Configuration** section. For example:

```markdown
## Loading your own configuration

It's possible to load your own configuration, which is useful if you want to connect to a remote cluster:

```bash
$ docker run --rm --name kubectl -v /path/to/your/kube/config:/.kube/config bitnami/kubectl:latest
(closing backticks)
```

**Comment by alemorcuq** [comment]
Thank you very much for this, @ESSch, this is definitely something that should be documented. Would you mind changing it a bit to follow our standards?

**Comment by alemorcuq** [comment]
Merging.

**Pull request merged**